src/schema/posts.ts

```diff
@@ -1793,10 +1793,13 @@ export const resolvers: IResolvers<any, Context> = {
         }
 
         const existingPost: Pick<ArticlePost, 'id' | 'deleted' | 'visible'> =
-          await manager.getRepository(ArticlePost).findOne({
-            select: ['id', 'deleted', 'visible'],
-            where: [{ url: cleanUrl }, { canonicalUrl: cleanUrl }],
-          });
+          await manager
+            .createQueryBuilder(Post, 'post')
+            .select(['post.id', 'post.deleted', 'post.visible'])
```

Comment:
I think you can omit `post` everywhere here right?

Reply:
I never get it to work without `post` or the alias that is set